app/views/components/_searchBar.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form_tag kondos_path, method: "GET" do %>
+<div class="d-flex justify-content-center">
+  <%= text_field_tag :search_kondos, params[:search_kondos], class:"form-control", placeholder: "Where are you?" %>
```

Comment:
We'll run into ```undefined method `where' for nil:NilClass``` in the `app/controllers/kondos_controller.rb` if the entered prefecture does not exist. I think this is fine for now and we can refine later. We can use select2 for this one later or find a better way to handle incorrect entry.

Reply:
Yeah I think the problem is in the controller `@kondos.where("LOWER(prefecture) like ?","%#{params[:search_kondos].to_s.downcase}%")` It doesn't find the invalid word in the Prefecture column. and you're right we can change it to select2 later